pkg/datapath/tables/route.go

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package tables
+
+import (
+	"fmt"
+	"net/netip"
+
+	"github.com/hashicorp/go-memdb"
+	"golang.org/x/sys/unix"
+
+	"github.com/cilium/cilium/pkg/statedb"
+)
+
+type Route struct {
+	Table     int
+	LinkName  string
+	LinkIndex int
+
+	Scope uint8
+	Dst   netip.Prefix
+	Src   netip.Addr
+	Gw    netip.Addr
+}
+
+func (r *Route) DeepCopy() *Route {
```

Comment:
Is this really a deepcopy? `r.Dst.{IP,Mask}`are still going to point to the underlying byte array of `r`.

Reply:
They're now `netip.Addr` and `netip.Prefix` so this is fixed.